Sources/AppCoinsSDK/Domain/Repositories/PayFlow/PayFlowRepository.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  PayFlowRepository.swift
+//
+//
+//  Created by Graciano Caldeira on 25/07/2024.
+//
+
+import Foundation
+
+internal class PayFlowRepository: PayFlowRepositoryProtocol {
+    
+    private let PayFlowService: PayFlowService = PayFlowClient()
+    
+    internal func setPayFlow() {
+        
+        let oemID = UserDefaults.standard.string(forKey: "attribution-oemid")
```

Comment:
Not the right way to access it, `PayFlowUseCases` should have access to `AttributionRepository`, call `getOEMID()` and pass it in the arguments of `setPayFlow()` to the repository